c/test/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@ else (CMAKE_CROSSCOMPILING)
     add_executable(test_libsbp check_main.c check_edc.c check_sbp.c ${generated_c_sources})
 
     target_link_libraries(test_libsbp ${TEST_LIBS})
+    set_property(TARGET test_libsbp PROPERTY C_STANDARD 99)
```

Comment:
nothing wrong with this, per se, but for maximum consistency, can we pick set_target_properties or set_properties(TARGET ...)?

Reply:
Fair enough, `set_target_properties` it is.